<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Descripteur</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>581</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Selectionner le numéro de l'image à modifier</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Auteur</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>330</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirmer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>280</y>
       <width>341</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>341</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>341</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>341</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Titre</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Taille*</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Coût*</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Extension</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>330</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>230</y>
       <width>341</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>341</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Image n°</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>370</y>
       <width>551</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>*: Merci de remplir ces cases pour qu'ils ne soient pas automatiquement remplacées par 0.</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>451</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Selectionner le numéro de l'image à supprimer</string>
    </property>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>251</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirmer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Image n°</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>451</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Entrer les valeurs min et max du coût .</string>
    </property>
    <widget class="QPushButton" name="pushButton_17">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirmer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Min</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_21">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>341</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_22">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>341</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Max</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>80</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string> Somme de coûts  €</string>
    </property>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 11pt &quot;Ubuntu&quot;;
gridline-color: rgb(32, 74, 135);
color: rgb(32, 74, 135);
border-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>210</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Somme de tailles Ko</string>
    </property>
    <widget class="QLCDNumber" name="lcdNumber_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 11pt &quot;Ubuntu&quot;;
gridline-color: rgb(32, 74, 135);
color: rgb(32, 74, 135);
border-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_backchoix">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>480</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>retour</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>811</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>266</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Importer</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Afficher</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Exporter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>511</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Auteur</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>300</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirmer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>250</y>
       <width>341</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>341</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>341</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>341</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Titre</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Taille</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Coût</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Extension</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>300</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>341</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>241</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Saisir les nouvelles données </string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 11pt &quot;URW Bookman L&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>groupBox_6</zorder>
   <zorder>tableWidget</zorder>
   <zorder>pushButton_backchoix</zorder>
   <zorder>groupBox</zorder>
   <zorder>label_15</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>994</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTri">
    <property name="title">
     <string>Trier</string>
    </property>
    <widget class="QMenu" name="menuTaille">
     <property name="title">
      <string>Taille</string>
     </property>
     <addaction name="actionD_croissant"/>
     <addaction name="actionD_croissant_2"/>
    </widget>
    <widget class="QMenu" name="menuCout">
     <property name="title">
      <string>Coût</string>
     </property>
     <addaction name="actionCroissant"/>
     <addaction name="actionD_croissant_3"/>
    </widget>
    <widget class="QMenu" name="menuTitre_Alph">
     <property name="title">
      <string>Titre (Ordre alphabétique)</string>
     </property>
     <addaction name="actionCroissant_2"/>
     <addaction name="actionD_croissant_4"/>
    </widget>
    <addaction name="menuTaille"/>
    <addaction name="menuCout"/>
    <addaction name="menuTitre_Alph"/>
   </widget>
   <widget class="QMenu" name="menuModifier">
    <property name="title">
     <string>Editer</string>
    </property>
    <widget class="QMenu" name="menuSupression">
     <property name="title">
      <string>Supprimer</string>
     </property>
     <addaction name="actionTable_enti_re"/>
     <addaction name="actionUn_descripteur"/>
    </widget>
    <addaction name="menuSupression"/>
    <addaction name="actionAjout"/>
    <addaction name="actionModifier"/>
   </widget>
   <widget class="QMenu" name="menuRechercher">
    <property name="title">
     <string>Rechercher</string>
    </property>
    <addaction name="actionCout_par_interval"/>
   </widget>
   <widget class="QMenu" name="menuCalculer">
    <property name="title">
     <string>Calculer</string>
    </property>
    <addaction name="actionCo_t_total"/>
    <addaction name="actionTaille_totale"/>
   </widget>
   <addaction name="menuTri"/>
   <addaction name="menuModifier"/>
   <addaction name="menuRechercher"/>
   <addaction name="menuCalculer"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionD_croissant">
   <property name="text">
    <string>Croissant</string>
   </property>
  </action>
  <action name="actionD_croissant_2">
   <property name="text">
    <string>Décroissant</string>
   </property>
  </action>
  <action name="actionCroissant">
   <property name="text">
    <string>Croissant</string>
   </property>
  </action>
  <action name="actionD_croissant_3">
   <property name="text">
    <string>Décroissant</string>
   </property>
  </action>
  <action name="actionCroissant_2">
   <property name="text">
    <string>De A à Z</string>
   </property>
  </action>
  <action name="actionD_croissant_4">
   <property name="text">
    <string>De Z à A</string>
   </property>
  </action>
  <action name="actionTable_enti_re">
   <property name="text">
    <string>Table entière</string>
   </property>
  </action>
  <action name="actionAjout">
   <property name="text">
    <string>Ajouter</string>
   </property>
  </action>
  <action name="actionCout_par_interval">
   <property name="text">
    <string>Coût (par intervalle)</string>
   </property>
  </action>
  <action name="actionModifier">
   <property name="text">
    <string>Modifier</string>
   </property>
  </action>
  <action name="actionUn_descripteur">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Un descripteur</string>
   </property>
  </action>
  <action name="actionTaille_totale">
   <property name="text">
    <string>Somme de tailles</string>
   </property>
  </action>
  <action name="actionCo_t_total">
   <property name="text">
    <string>Somme de coûts</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
